pkg/cmd/helmfile/move/move.go

```diff
@@ -330,25 +327,33 @@ func (o *Options) moveFilesToClusterOrNamespacesFolder(dir, ns, releaseName, cha
 	return nil
 }
 
-func (o *Options) isClusterWide(kind string, apiVersion string, client versioned.Interface) (bool, error) {
+func (o *Options) isClusterWide(kind string) (bool, error) {
 	if kube.IsNoKubernetes() {
 		// Approximates the truth
 		return kyamls.IsClusterKind(kind), nil
 	}
-	val, ok := o.ClusterWide[kind]
-	if !ok {
-		apiResourceList, err := client.Discovery().ServerResourcesForGroupVersion(apiVersion)
+	if o.ClusterWide == nil {
+		o.ClusterWide = make(map[string]bool)
+		client, err := jxclient.LazyCreateJXClient(nil)
 		if err != nil {
-			return true, err
+			return kyamls.IsClusterKind(kind), errors.Wrapf(err, "Failed to create k8s client")
 		}
-
-		for _, resource := range apiResourceList.APIResources {
-			o.ClusterWide[resource.Kind] = !resource.Namespaced
+		apiResourceLists, err := client.Discovery().ServerPreferredResources()
+		if err != nil {
+			return kyamls.IsClusterKind(kind), errors.Wrapf(err, "Failed to fetch api resources")
```

Comment:
Use fmt.Errorf for consistency? Also pkg/errors is unmaintained ....

Reply:
errors.Wrapf is used much more than fmt.Errorf in this file. So, no, that would not make it more consistent.

Reply:
I meant in ur change, u used fmt.Errorf, so if you can use that instead of errors.Wrapf, then that would be good. May be new code should try to reduce using errors package ...

Reply:
Well, I can't since fmt.Errorf can't be used to wrap an error.